telegram/ext/filters.py

```diff
@@ -25,11 +25,12 @@
 
 from telegram import Chat, Update, MessageEntity
 
-__all__ = ['Filters', 'BaseFilter', 'InvertedFilter', 'MergedFilter']
+__all__ = ['Filters', 'BaseFilter', 'MessageFilter', 'UpdateFilter', 'InvertedFilter',
+           'MergedFilter']
 
 
 class BaseFilter(ABC):
-    """Base class for all Message Filters.
+    """Base class for all Filters.
 
     Subclassing from this class filters to be combined using bitwise operators:
```

Comment:
I know you didn't change this line, but I just realised it makes like zero sense. Could you maybe fix that? :P

Reply:
Upsie, didn't even see that :D. Fixed